expense/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "@tsconfig/node14/tsconfig.json",
+  "version": "4.2.2",
+  "compilerOptions": {
+    "sourceMap": true,
+    "esModuleInterop": true,
```

Comment:
This is already true when using `@tsconfig/node14`: https://www.npmjs.com/package/@tsconfig/node14